CMakeLists.txt

```diff
@@ -18,12 +18,19 @@ set( BUILD_SHARED_LIBS ${SFGUI_BUILD_SHARED_LIBS} )
 option( SFGUI_BUILD_EXAMPLES    "Build examples."                               ON)
 option( SFGUI_BUILD_DOC         "Generate API documentation."                   OFF)
 option( SFGUI_INCLUDE_FONT      "Include default font in library (DejaVuSans)." ON)
-option( SFML_STATIC_LIBRARIES   "Do you want to link SFML statically?"          OFF)
 
 
 # Find packages.
 find_package( OpenGL REQUIRED )
-find_package( SFML 2.5 REQUIRED COMPONENTS graphics window system )
+if ( EXISTS ${CMAKE_SOURCE_DIR}/conanbuildinfo.cmake )
+	include( ${CMAKE_SOURCE_DIR}/conanbuildinfo.cmake )
+	conan_basic_setup()
+else ()
+	option( SFML_STATIC_LIBRARIES   "Do you want to link SFML statically?"          OFF)
+	find_package( SFML 2.5 REQUIRED COMPONENTS graphics window system )
+	set( SFML_LIBRARIES sfml-graphics sfml-window sfml-system )
```

Comment:
With the new config files, you don't need to specify all the targets, but it's enough to specify `sfml-graphics`.

Reply:
Great! I'll update that when I get a chance.